test/org/javarosa/xform/parse/XFormParserTest.java

```diff
@@ -386,6 +386,11 @@ public void parseFormWithSetValueAction() throws IOException {
         assertThat(groupElement.getBind(), is(expectedXPathReference));
     }
 
+    @Test public void parseWithWhitelistedAttributes() throws IOException {
```

Comment:
When I created this class last April,

https://github.com/dcbriccetti/javarosa/commit/ab7517e984a34bedf82ddfe15507d7e4dc3381f2

I established a method naming convention based on the indicative mood (“parses…”, rather than “parse…”), in order to express what each method tests. I carried on like this for months until @mdudzinski joined in and started littering the code with imperative mood method names. :wink: I think he is not a native English speaker so he deserves some leeway. Now, @daveycrockett, you must choose which of us you wish to align with. :grin: